code-challenges.test.js

```diff
@@ -21,36 +22,123 @@
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// Pseudo Code
+// create a test for the expected outputs
+// create a describe statement with new function name
+// create let statements for each example inputs
+// create an it statement that states the purpose of our function
+// create expect statements for each example outputs
+
+describe("fibSeq", () => {
+  let newSeq = [1, 1, 2, 3, 5, 8];
+  let fullerSeq = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55];
+  it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence", () => {
+    expect(fibSeq(6)).toEqual([1, 1, 2, 3, 5, 8]),
+      expect(fibSeq(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55]);
+  });
+});
 
 // b) Create the function that makes the test pass.
 
+// Pseudo Code
+// create a const for you above describe statement
+// assign newArr as a variable
+// create a for loop
+// use .push to add newArr
+// create return for newArr
+
+const fibSeq = (num) => {
+  let newArr = [1, 1];
+  for (let i = 1; i < num - 1; i++) {
+    newArr.push(newArr[i] + newArr[i - 1]);
+  }
+  return newArr;
+};
 
 // --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"];
 // Expected output: [-9, 7, 9, 199]
 
-const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"];
 // Expected output: [-823, 7, 23]
 
+// Pseudo Code
+// create a test for the expected outputs
+// create a describe statement with new function name
+// create let statements for each example inputs
+// create an it statement that states the purpose of our function
+// create expect statements for each example outputs
+
+describe("newArr", () => {
+  let fullArr = [-9, 7, 9, 199];
+  let fullerArr = [-823, 7, 23];
+  it("pulls specific values from the above constants", () => {
+    expect(newArr(fullArr1)).toEqual(fullArr);
+    expect(newArr(fullArr2)).toEqual(fullerArr);
+  });
+});
 
 // b) Create the function that makes the test pass.
 
+// Pseudo Code
+// create a function named newArr
+// use .filter calling upon a value
+// return only odd numbers
+// sort arrays a and b
+
+const newArr = (array) => {
+  let result = array.filter((value) => {
+    return typeof value === "number" && value % 2 !== 0;
+  });
+  return result.sort((a, b) => a - b);
+};
```

Comment:
Logic looks excellent!  I would recommend naming your function with a more descriptive name to the problem you are solving.  This will really help as you build larger applications to easily determine what the function's purpose is.